suitcase/pizza_box/_version.py

```diff
@@ -43,7 +43,7 @@ def get_config():
     cfg.style = "pep440-post"
     cfg.tag_prefix = "v"
     cfg.parentdir_prefix = "None"
-    cfg.versionfile_source = "suitcase/suitcase_pizza-box/_version.py"
+    cfg.versionfile_source = "suitcase/suitcase_pizza_box/_version.py"
```

Comment:
Should it still be `"suitcase/suitcase_pizza_box/_version.py"`, not `"suitcase/pizza_box/_version.py"`?

Reply:
thanks, i guess this needs to be changed  [here](https://github.com/NSLS-II/suitcase-cookiecutter/blob/master/suitcase-%7B%7B%20cookiecutter.subproject_name%20%7D%7D/suitcase/%7B%7B%20cookiecutter.subpackage_name%20%7D%7D/_version.py#L46)

@danielballan can you quickly update that and regenerate?

Reply:
since it doesnt take long and i was here did it [here](https://github.com/NSLS-II/suitcase-pizza-box/pull/3). thanks @mrakitin !

Reply:
Welcome 😃